### DISCUSSION TITLE
I can't change the main application form.

### DISCUSSION THREAD

**Discussion by jeanssp** [dotnet/maui#20921] [General]
![2024-02-29_160231](https://github.com/dotnet/maui/assets/65717945/60995e94-e9fb-408c-bb73-80ba1409cbc8)
![2024-02-29_160317](https://github.com/dotnet/maui/assets/65717945/be87025a-1c31-47e5-a990-7a0b9e0ae483)

<ContentPage
 x:Class="Maui11082023.Mvvm.View.AccountsPage"

    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             
    xmlns:model="clr-namespace:Maui11082023.Mvvm.Models"
    xmlns:viewmodel="clr-namespace:Maui11082023.Mvvm.ViewModel"
    xmlns:ios="clr-namespace:Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific;assembly=Microsoft.Maui.Controls"
    ios:Page.UseSafeArea="True"
    Title="Счета"
    x:DataType="viewmodel:AcctViewModel">
    <Grid
        ColumnDefinitions="*,*"
        ColumnSpacing="5"
        RowDefinitions="*,Auto"
        RowSpacing="0"
        BackgroundColor="{AppThemeBinding Light={StaticResource LightBackground}, Dark={StaticResource DarkBackground}}">
        <RefreshView
            Grid.ColumnSpan="2"
            
            Command="{Binding GetAcctsCommand}"
            
            IsRefreshing="{Binding IsRefreshing}">
            <ContentView>
                <CollectionView
                        ItemsSource="{Binding Accts}"
                        SelectionMode="None">
                    <CollectionView.EmptyView>
                        <StackLayout Padding="100">
                            <Image
                                    HorizontalOptions="Center"
                                    HeightRequest="160"
                                    WidthRequest="160"
                                    Source="nodata.png"
                                    VerticalOptions="Center" />
                        </StackLayout>
                    </CollectionView.EmptyView>
                    
                    <CollectionView.ItemTemplate>
                        <DataTemplate x:DataType="model:AcctDto">
                            <Grid Padding="10" HorizontalOptions="Fill">
                                <Frame HeightRequest="125" Style="{StaticResource CardView}">
                                    <Frame.GestureRecognizers>
                                        <TapGestureRecognizer 
                                                Command="{Binding Source={RelativeSource AncestorType={x:Type viewmodel:AcctViewModel}}, Path=GoToDetailsCommand}"
                                                CommandParameter="{Binding .}"/>
                                    </Frame.GestureRecognizers>
                                    <Grid Padding="0" ColumnDefinitions="125,*">
                                        <Image
                                                Aspect="AspectFill"
                                                HeightRequest="125"
                                                Source="{Binding AcctName}"
                                                WidthRequest="125" />
                                        <VerticalStackLayout
                                                VerticalOptions="Center"
                                                Grid.Column="1"
                                                Padding="10">
                                            <Label Style="{StaticResource LargeLabel}" Text="{Binding  DebetSaldo}" />
                                            <Label Style="{StaticResource MediumLabel}" Text="{Binding CreditSaldo}" />
                                        </VerticalStackLayout>
                                    </Grid>
                                </Frame>
                            </Grid>
                        </DataTemplate>
                    </CollectionView.ItemTemplate>
                </CollectionView>
            </ContentView>
        </RefreshView>

        <Button
            Grid.Row="1"
            Grid.Column="0"
            Margin="8"
            Command="{Binding GetAcctsCommand}"
            IsEnabled="{Binding IsNotBusy}"
            Style="{StaticResource ButtonOutline}"
            Text="Get Monkeys" />

        <ActivityIndicator
            Grid.RowSpan="2"
            Grid.ColumnSpan="2"
            HorizontalOptions="Fill"
            IsRunning="{Binding IsBusy}"
            IsVisible="{Binding IsBusy}"
            Color="{StaticResource Primary}"
            VerticalOptions="Center" />
    </Grid>
</ContentPage>

**Comment by davidortinau** [comment]
Definitely odd. Did you figure it out? 

To troubleshoot this I would usually start adding bright colors to the background of the layout elements all the way to the ContentPage, and see how things are actually sizing.

I don't think you need that `ContentView` wrapping the `CollectionView` fwiw.
